File: ios/Classes/model/InstantMessagingAccount.swift
//
//  InstantMessagingAccount.swift
//  contacts
//
//  Created by Maurits van Beusekom on 28/08/2018.
//

import Foundation

class InstantMessagingAccount : Codable {
    var account : String = ""
    var label : String = ""
    var service : InstantMessagingService = InstantMessagingService.other
}

File: android/src/main/kotlin/com/baseflow/contactsplugin/models/InstantMessagingAccount.kt
package com.baseflow.contactsplugin.models

import com.baseflow.contactsplugin.data.InstantMessagingService

class InstantMessagingAccount {
    var account: String = ""
    var label: String = ""
    var service: InstantMessagingService = InstantMessagingService.OTHER
}